x-pack/filebeat/cmd/root.go

```diff
@@ -0,0 +1,10 @@
+package cmd
```

Comment:
And I assume that the top-level `make check` fails because these files do not have Apache headers (I didn't check). That can be addressed by merging https://github.com/elastic/go-licenser/pull/14 and using the new -exclude flag.

Reply:
I opened https://github.com/elastic/go-licenser/pull/15 to handle this, will update this PR once that's merged

Reply:
Thanks for adding this @exekias  I just created https://github.com/elastic/beats/pull/7837 which leverage your work :)

Reply:
Personally I use [beatsfmt](https://github.com/urso/beatsfmt). This one searches for a `.go_license_header` file relative to the source file to be formatted.